### DISCUSSION TITLE
@set in defer mode

### DISCUSSION THREAD

**Discussion by jamesRUS52** [livewire/livewire#2326] [3. Feature ideas]
Hi, I am using select2 component and to change value of binded variable I use this trick

```
<script>
        document.addEventListener("livewire:load", function (event) {
            $('select').on('change', function (event) {
                @this.set(event.target.id, $(this).val());
            });
        });
</script>
```

But this fires immidiatly after value changed in select2 component and refresh the livewire component.
Is the way do somethink like this, but in defer mode, like wire:model.defer ?

**Comment by joshhanley** [comment]
@jamesRUS52 yeah you can pass a third parameter to set, which is a boolean for defer, so set it to true to defer.

So the signature of the method is
```js
set(name, value, defer = false)
```

**Reply by jamesRUS52** [reply]
Thank you, it work fine
